eth2_testgen/bls/tgen_bls.py

```diff
@@ -0,0 +1,188 @@
+"""
+BLS test vectors generator
+Usage:
+    "python tgen_bls path/to/output.yml"
+"""
+
+# Standard library
+import sys
+from typing import Tuple
+
+# Third-party
+import yaml
+
+# Ethereum
+from eth_utils import int_to_big_endian, big_endian_to_int
+
+# Local imports
+import bls
+from hash import hash_eth2
+
+
+def int_to_hex(n: int) -> str:
+    return '0x' + int_to_big_endian(n).hex()
+
+
+# Note: even though a domain is only an uint64,
+# To avoid issues with YAML parsers that are limited to 53-bit (JS language limit)
+# It is serialized as an hex string as well.
+DOMAINS = [
+    0,
+    1,
+    1234,
+    2**32-1,
+    2**64-1
+]
+
+MESSAGES = [
+    b'message',
+    b'Bigger message',
+    b'Very .............. long ............. message .... with entropy: 1234567890-beacon-chain'
+]
+
+PRIVKEYS = [
+    # Private keys are 48 bytes, Keccack hashing only produces 32 bytes
+    # So we concat two and slice
+    big_endian_to_int((hash_eth2(b'Alice') + hash_eth2(b'ecilA'))[:48]),
+    big_endian_to_int((hash_eth2(b'Bob') + hash_eth2(b'boB'))[:48]),
+    big_endian_to_int((hash_eth2(b'Eve') + hash_eth2(b'evE'))[:48])
```

Comment:
Private keys must be lower than curve order '0x73eda753299d7d483339d80809a1d80553bda402fffe5bfeffffffff00000001'

The best would be to take the modulo but as I'm not sure any integers lower than than the curve order is valid, I hardcoded the following private key values:

```python
PRIVKEYS = [
    # Curve order is 256 so private keys are 32 bytes at most.
    # Also not all integers is a valid private key, so using pre-generated keys
    hex_to_int('0x00000000000000000000000000000000263dbd792f5b1be47ed85f8938c0f29586af0d3ac7b977f21c278fe1462040e3'),
    hex_to_int('0x0000000000000000000000000000000047b8192d77bf871b62e87859d653922725724a5c031afeabc60bcef5ff665138'),
    hex_to_int('0x00000000000000000000000000000000328388aff0d4a5b7dc9205abd374e7e98f3cd9f3418edb4eafda5fb16473d216'),
]
```

This needs a int_to_hex proc defined as follow:

```python
def hex_to_int(x: str) -> int:
    return int(x, 16)
```